app/models/evaluation.rb

```diff
@@ -67,4 +69,45 @@ def self.filters_to_json
       }
     ].to_json
   end
+
+  def self.to_csv(ids = [])
+
+    csv_string = CSV.generate do |csv_line|
+
+    evaluation_columns = Evaluation.new.attributes.keys
+    evaluation_columns << "evaluation_id"
+
+    excluded_attributes = ["created_at", "updated_at", "id", "site_id", "source_id"]
+
+    evaluation_columns.delete_if { |k, v| excluded_attributes.include? k }
+
+    additional_columns = ["wpda_id", "iso3", "name", "designation", "source_data_title", "source_resp_party", "source_year", "source_language"]
+    evaluation_columns << additional_columns.map{ |e| "#{e}" }
```

Comment:
I'm not even sure why I need this line, but it prevents the quotations appearing inside the csv!